apps/user-office-frontend/src/components/proposal/ParticipantModal.tsx

```diff
@@ -30,6 +30,7 @@ function ParticipantModal(props: {
   userRole?: UserRole;
   invitationUserRole?: UserRole;
   participant?: boolean;
+  setPi?: (user: BasicUserDetails) => void;
```

Comment:
It would be nice to be more descriptive with the setPi to setPrincipalInvestigator